### PR TITLE
CHANGELOG: fix typo

### PR DESCRIPTION
**Pull request by chymy**
Signed-off-by: chymy <chang.min1@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.3.md [hunk 1]
```diff
@@ -439,8 +439,7 @@ Kubectl flag `--container-port` flag is deprecated: it will be removed in the fu
   * ARP caching fix
   * Use /dev/xvdXX names
   * ELB:
-    * ELB proxy protocol support
-    * mixed plaintext/encrypted ports support in ELBs
+    * ELB proxy protocol support mixed plaintext/encrypted ports support in ELBs
```
Comment (hasheddan) [inline CHANGELOG/CHANGELOG-1.3.md:442]: I’m not sure they points actually should be condensed here

**Comment by chymy** [comment]
Thanks for your reviews @hasheddan ， I hava fixed.

**Review by feiskyer — APPROVED** [review]
/lgtm
/ok-to-test

**Comment by chymy** [comment]
/assign @liggitt
Thanks!

**Pull request merged**